src/cryptoadvance/specter/templates/includes/overlay/sign_tx_method.jinja

```diff
@@ -62,7 +62,7 @@
         {% endif %}
         {% if device.hot_wallet %}
             <button id="{{ device.alias }}_hot_sign_btn" class="btn flex-item" style="width: 190px; margin: 15px auto;">
-                {% if device.device_type == "bitcoincore" %}
+                {% if device.device_type == "bitcoincore" or device.device_type == "bitcoincore_watchonly" %}
```

Comment:
Hmm this probably should be treated differently as a watch only can't sign

Reply:
right, maybe i think we should not render the button for signing the transaction in this case.